outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "vpc_id" {
+  value = "${data.aws_vpc.default.id}"
+}
+
+output "bucket_name" {
+  value = "${aws_s3_bucket.logs.bucket_domain_name}"
+}
+
+output "backups_domain_name" {
+  value = "${aws_s3_bucket.backups.bucket_domain_name}"
+}
+
+output "efs_backups_id" {
+  value = "${aws_s3_bucket.backups.id}"
+}
+
+output "datapipeline_ids" {
+  value = "${aws_cloudformation_stack.datapipeline.*.outputs["DataPipelineId"]}"
+}
+
+output "sg_for_efs_instances" {
```

Comment:
It is not possible, because we should add this security group to EFS instances.
We do not have an ability to add an SG to existing EFS by TF resources.
We should add these SG manually.